programs/sll_sum_lists_forward.py

```diff
@@ -0,0 +1,66 @@
+# Sum lists forward order (page 95. 2.5)
+# input 2 sll exem. (6 -> 1 -> 7)  (2 -> 9 -> 5)   its equal to 617+295 = 912
+# return the sum as a sll ( 9 -> 1 -> 2)
+
+from single_linked_lists import Node
+
+
+def sum_nodes(a: Node, b: Node):
+    head_a = a
+    head_b = b
+    len_a = 0
+    len_b = 0
+
+    # compare len of the 2 sll
+    while a or b:
+        if a:
+            a = a.next
+            len_a += 1
+        if b:
+            b = b.next
+            len_b += 1
+
+    # add 0 to the smaller sll
+    if (len_a - len_b) < 0:
```

Comment:
Dont repeat yourself. 

Swap the pointers so that the bigger list is always the same. By doing that, lines 30 to 34 can be removed

Also, this condition is overly complicated, it is easier just to do `len_a < len_b`